### PR TITLE
Blog Post: World Ocean Day

### PR DESCRIPTION
**Pull request by steffilazerte**
Draft
* [X] I have read the Content Guidelines.
* [X] I have read the Technical Guidelines.
* [X] I used or followed the R Markdown or Markdown template.
* [X] I have followed the Style Guide.
* [X] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags (incuding "community" tag).
* [X] I have added the "tech notes" tag if this is a technote.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.

### CODE REVIEW

**Comment by steffilazerte** [comment]
Okay @stefaniebutland, I'm ready for a review! I had to switch out robis in the end, for rinat because robis isn't actually rOpenSci 🤦🏼 

@sckott could you do a quick review too? For the use of your rnoaa?

The second section might be a bit wishy washy as I changed things over last minute (I need some space from this for a while!)

Thanks!

**Comment by sckott** [comment]
Looks good @steffilazerte 

- Just saw this the other day from mike sumner https://github.com/AustralianAntarcticDivision/seaice maybe worth mentioning in notes, up to you
- mike references in the above repo this ropensci blog post https://ropensci.org/blog/2018/11/13/antarctic/ that talks about sea ice data

**Review by sckott — APPROVED** [review]

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+> -- About World Ocean Day - <https://worldoceanday.org/about/>
+
+This year has been hard, and like many, I often feel isolated and emotionally exhausted.
+The idea that we have only one ocean and it connects us is a lovely way to remember that as isolated as we may feel we're all here on this planet together.
+
+So in honour of World Ocean Day, let's take a look at some important ocean metrics using open data and open software via rOpenSci packages!
+
+## Sea ice
+
+One of the biggest challenges for protection our ocean is climate change[^2].
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:86]:
```suggestion
One of the biggest challenges for protecting our ocean is climate change[^2].
```

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+Last September the Arctic sea ice had the 2nd-lowest extent in a 40-year record[^3]. 
+This isn't good for the Arctic ecosystem and has impacts not only on wildlife but on humans as well [^4]. `r emo::ji("sad")`
+
+Let's see if we can't explore ice extent ourselves using open data and open source packages part of the [rOpenSci package collections](/packages).
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:90]:
```suggestion
Let's see if we can't explore ice extent ourselves using open data and open source packages that are part of the [rOpenSci package collections](/packages).
```

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+
+Here, we'll use two rOpenSci packages
+- [rnoaa](https://docs.ropensci.org/rnoaa) to access NOAA data on sea ice coverage
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:93]: can you add somewhere, something like
rnoaa by Scott Chamberlain for rOpenSci and rnaturalearth by Andy South, peer reviewed by rOpenSci

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+- and [rnaturalearth](https://docs.ropensci.org/rnaturalearth) to get coastal outlines for context
+
+(Both these packages can do so much more, so check out the docs if you're curious)
+
+We'll also use sf for working with spatial data, dplyr for data manipulation, and ggplot2 for plotting.
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:98]: pondering whether it's good to link to these pkgs

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+```
+
+Let's get some ice data! 
+We'll use the `sea_ice()` function to grab data on ice extent for every 5 years from 1980 to 2020 (`year = seq(1980, 2020, 5)`), in September (`month = "Sep"`), and for the Arctic (North) (`pole = "N"`).
+
+```{r, message = FALSE, cache = TRUE}
+ice <- sea_ice(year = seq(1980, 2020, 5), month = "Sep", pole = "N")
+```
+
+`ice` is a list, which each list item corresponding to a year.
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:116]:
```suggestion
`ice` is a list, with each list item corresponding to a year.
```

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+Let's take a brief look at what we've got, by scanning the `head()` of the first two years worth of data. 
+```{r}
+head(ice[[1]])
+head(ice[[2]])
+```
+
+Each year is a 'fortified' data frame which you could plot using `geom_polygon()`. 
+But I want to get the projection right, so let's bind the rows together (adding year as a parameter) and convert to an sf object. 
+We'll transform to EPSG 3411 which refers to the NSIDC Sea Ice Polar Stereographic North projection commonly used to map the Arctic.
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:125]: can you link to this for folks unfamiliar?

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,333 @@
...
+
+Let's take a peak at what we've got
+
+```{r}
+ggplot() +
+  theme_minimal() +
+  geom_sf(data = ice_sf, aes(fill = year), colour = NA)
+```
+
+Oooo very nice! But a couple of points could be improved
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:160]: I'd appreciate ~ 1 sentence to help reader (me) quickly interpret what the map/plot is telling me

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,333 @@
...
+
+So in honour of that wonderful story, let's do another exploration of the ocean, but this time by looking at leopard seals in Antarctica! 
+
+<!--html_preserve-->
+{{< figure src = "leopard_seal.jpg" class = "center" alt = "Leopard seal lying on the ice. The seal is grey on the back, white on the belly with grey speckles">}}
+<!--/html_preserve-->
+*[Andrew Shiva / Wikipedia / CC BY-SA 4.0](https://commons.wikimedia.org/wiki/File:Antarctic_Sound-2016-Brown_Bluff%E2%80%93Leopard_seal_(Hydrurga_leptonyx)_04.jpg)*
+
+
+In addition to rnaturalearth, sf, dplyr, and ggplot2 which we loaded above, we'll use lubridate to handle date/times, and the rOpenSci package [rinat](https://docs.ropensci.org/rinat), for accessing species observations from the citizen science project [iNaturalist](https://www.inaturalist.org/).
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:226]: again pls mention pkg author/maintainer 
and tag folks when you tweet

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,333 @@
...
+```{r, cache = TRUE}
+seals <- get_inat_obs(taxon_name = "leopard seal") %>%
+  select(scientific_name, datetime, description, latitude, longitude) %>%
+  filter(datetime != "") %>%
+  mutate(datetime = ymd_hms(datetime),
+         year = year(datetime))
+glimpse(seals)
+```
+
+Alright, we have 59 observations around Antarctica, let's see where they're from.
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:245]:
```suggestion
Alright, we have 59 observations of leopard seals around Antarctica, let's see where they're from.
```

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 10]
```diff
@@ -0,0 +1,333 @@
...
+- [2018-12-04 rOpenSci blog "rnoaa: new data sources and NCDC units"](/blog/2018/12/04/rnoaa-update/)
+- [Use cases](https://discuss.ropensci.org/search?q=rnoaa%20category%3A10)
+
+**[rnaturalearth](https://docs.ropensci.org/rnaturalearth)**
+
+- [Use cases](https://discuss.ropensci.org/search?q=rnaturalearth%20category%3A10)
+
+**[rinat](https://docs.ropensci.org/rinat)**
+
+- [2014-03-26 rOpenSci blog "Accessing iNaturalist data](/blog/2014/03/26/rinat/)
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:315]:
```suggestion
- [2014-03-26 rOpenSci blog "Accessing iNaturalist data"](/blog/2014/03/26/rinat/)
```

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 11]
```diff
@@ -0,0 +1,333 @@
...
+{{< figure src = "https://raw.githubusercontent.com/allisonhorst/palmerpenguins/master/man/figures/lter_penguins.png" alt = "Cartoons of Chinstrap, Gentoo, and Adelie penguins" width = "500" class = "center">}}
+<!--/html_preserve-->
+*Artwork by [Allison Horst](https://github.com/allisonhorst/palmerpenguins)*
+
+
+## Learn more about these awesome packages
+
+**[rnoaa](https://docs.ropensci.org/rnoaa)**
+
+- [2014-03-13 rOpenSci blog "rnoaa - Access to NOAA National Climatic Data Center data"](/blog/2014/03/13/rnoaa/)
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:305]: list newer post first?

Comment (stefaniebutland) [reply content/blog/2021-06-08-world-ocean-day/index.Rmd:305]: If you see any challenges with linking to such old posts, maybe note that they're old?

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 12]
```diff
@@ -0,0 +1,333 @@
...
+
+**[rnaturalearth](https://docs.ropensci.org/rnaturalearth)**
+
+- [Use cases](https://discuss.ropensci.org/search?q=rnaturalearth%20category%3A10)
+
+**[rinat](https://docs.ropensci.org/rinat)**
+
+- [2014-03-26 rOpenSci blog "Accessing iNaturalist data](/blog/2014/03/26/rinat/)
+
+## Have your own use cases?
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:317]:
```suggestion
## Have you used rOpenSci packages to study our ocean?
```

File: content/blog/2021-06-08-world-ocean-day/index.Rmd [hunk 13]
```diff
@@ -0,0 +1,333 @@
...
+We'd love to share them!
```
Comment (stefaniebutland) [inline content/blog/2021-06-08-world-ocean-day/index.Rmd:318]:
```suggestion
We'd love to share your examples!
```

**Review by stefaniebutland — COMMENTED** [review]
I LOVE the post. tone, flow, technical but inviting to anyone.

this is brilliant esp with the image after
Screen Shot 2021-06-07 at 9 25 43 AM


Before the call to action for use cases could you include a list of other rOpenSci pkgs that can/have been used to study ocean stuff?
worrms, mregions, https://docs.ropensci.org/antanym/

And and add a sentence (similar spot near end) that this post by Ben Raymond and Michael Sumner on behalf of the Antarctic rOpenSci community shares a use case ....blah blah sea ice and ocean depth ... shows the path of an elephant seal ....https://ropensci.org/blog/2018/11/13/antarctic/

File: content/blog/2021-06-08-world-ocean-day/index.Rmd (hunk 4 above)
Comment (steffilazerte) [reply content/blog/2021-06-08-world-ocean-day/index.Rmd]: Done

File: content/blog/2021-06-08-world-ocean-day/index.Rmd (hunk 11 above)
Comment (steffilazerte) [reply content/blog/2021-06-08-world-ocean-day/index.Rmd]: I changed the order, and didn't mention problems. That's why the dates are there, is should (hopefully) be apparent

File: content/blog/2021-06-08-world-ocean-day/index.Rmd (hunk 3 above)
Comment (steffilazerte) [reply content/blog/2021-06-08-world-ocean-day/index.Rmd]: Done

**Comment by steffilazerte** [comment]
Thanks @stefaniebutland and @sckott 

In addition the the specific comments above:
- I added references to the seaice package
- I added a new section for other rOpenSci resources including the blog post and other packages

**Comment by steffilazerte** [comment]
Draft Tweet

---

[blog] 🎉 Celebrating World Ocean Day rOpenSci Style 🎉 by @steffilazerte

🐠🌊🐬🦭🐙🐟 Today is World Ocean Day! 🐟🐙🦭🐬🌊🐠

Let's celebrate and honour the Ocean which connects us all by exploring some themes with rOpenSci packages

- 🧊 Arctic sea ice changes over the years (#rnaturalearth and #rnoaa)
- 🦭Leopard seal observations in the Antarctic from iNaturalist (#rnaturalearth and #rinat)

https://ropensci.org/blog/2021/06/08/world-ocean-day/

#rstats #worldoceanday

(1/2)

---

🙏 Thanks to the awesome package authors and maintainers:
@southmapr (rnaturalearth), @sckottie (rnoaa for rOpenSci), Edmund Hart and Stéphane Guillou (rinat)

(2/2)

**Pull request merged**